pixels/endpoints/moderation.py

```diff
@@ -177,7 +177,7 @@ async def webhook(request: Request) -> Message:
                 log.warning(f"Non 200 status code from Discord: {edit_resp.status_code}\n{edit_resp.text}")
                 last_message_id = None
 
-        # If no message is found in cache or message is missing, create new message
+        # If no message is found in cache, message is missing or edit failed, create new message
```

Comment:
Actually this could probably be improved even more
```suggestion
        # If no message is found in cache, the message is missing or the edit failed, send a new message
```